### PR TITLE
onefetch: Update to version 2.5.0

### PR DESCRIPTION
**Pull request by o2sh**

### CODE REVIEW

File: bucket/onefetch.json [hunk 1]
```diff
@@ -1,21 +1,20 @@
 {
-    "version": "2.2.0",
-    "description": "A command-line system information tool that displays information about your project directly on your terminal.",
+    "version": "2.5.0",
+    "description": "Git repository summary on your terminal.",
     "homepage": "https://github.com/o2sh/onefetch",
     "license": "MIT",
-    "notes": "You may have to enable ANSI Color Codes in your terminal. See https://stackoverflow.com/questions/51680709/",
```
Comment (Ash258) [inline bucket/onefetch.json:6]: Why did you delete this?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/onefetch.json (hunk 1 above)
Comment (o2sh) [reply bucket/onefetch.json:6]: It's handled in the [code](https://github.com/o2sh/onefetch/blob/master/src/main.rs)

```rust
#[cfg(windows)]
let _ = ansi_term::enable_ansi_support();
```
